monitoring/rid_qualifier/host/README.md

```diff
@@ -1,9 +1,38 @@
+# RID Qualifier Host
+
 `rid_qualifier host` is a development-level web server that provides endpoint to initialize the automated testing of the rid qualifier. 
 This PR includes the initial setup for the web-server that can be run locally on your system via the (run_locally.sh)[run_locally.sh] script. 
 
-**Prerequisite**: Local DSS instance should be started by running (monitoring/rid_qualifier/mock/run_locally.sh)[monitoring/rid_qualifier/mock/run_locally.sh] script.
+## About
+
+This project will create the following environment:
+
+* redis Container: A redis server to handle background tasks.
+
+* rq-worker Container: A container to run redis queue worker processes.
+
+* rid-host Container: A flask application which accepts flight states' files, collect auth and config spec from user and executes RID Qualifier task. Once task finishes successfully, report.json file is available to download.
+
+## Run via Docker
+
+You should first create a new Python virtual environment to install the
```

Comment:
Is there a need to set up/activate an appropriate virtual environment if Docker is going to be used?  Generally, the appropriate virtual environment is built inside the Docker images so it doesn't need to be built on the host machine, and the run_locally.sh file below just uses Docker